src/main.go

```diff
@@ -38,17 +39,19 @@ var (
 	defaultScrapeTime           = time.Second * 3
 	systemMetricsPath           = "storage/system.json"
 	journalMetricsPath          = "storage/journal.json"
+	filesPath                   = "storage/000000"
```

Comment:
This shouldn' be hardcoded. also, the `000000` should be kept as int and can be convertable to string so that we can do arithmetic operations on it.

Reply:
I'd like to know the motivation behind this pathname?

Reply:
There is no particular reason for that. I have just named in for development purpose, we can change it later.